src/Contrib/Grpc/GrpcTransportFactory.php

```diff
@@ -75,11 +82,25 @@ public function create(
         return new GrpcTransport(
             $grpcEndpoint,
             $opts,
-            $method,
+            GrpcTransport::method($this->signal),
             $headers,
         );
     }
 
+    public function withSignal(string $signal): self
```

Comment:
Should be moved to `OpenTelemetry\Contrib\Otlp`.

Reply:
removed altogether as part of removing OtlpTransportFactoryInterface